pkg/schema/schema.go

```diff
@@ -785,9 +785,27 @@ func populateParts(m map[string]interface{}, size int64, parts []BytesPart) erro
 }
 
 func newBytes() *Builder {
-	return base(1, "bytes")
+	return base(1, BytesCamliType)
 }
 
+// CamliType is one of the valid "camliType" fields in a schema blob. See doc/schema
+type CamliType string
+
+const (
+	BytesCamliType     CamliType = "bytes"
```

Comment:
How about "TypeBytes", "TypeClaim", etc, dropping "Camli" entirely from at least the const name?

Reply:
Looks good, done.